atomicapp/nulecule/base.py

```diff
@@ -266,11 +269,13 @@ def __init__(self, name, basepath, source=None, params=None,
         self.rendered_artifacts = defaultdict(list)
         self._app = None
         self.config = config
+        self.display = Display()
```

Comment:
The NuleculeBase base class has `self.display = Display()`. Why do we need to redefine it here?